keps/prod-readiness/sig-storage/2924.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2924
+alpha:
+  approver: "@ehashman"
```

Comment:
@wojtek-t is looking at all csi migration related keps this cycle.

Reply:
@msau42 updated , ptal..thanks